### PR TITLE
Updating tplink docs to include missing dimmer option.

### PR DESCRIPTION
**Pull request by vangorra**
**Description:**
Updating documentation to include dimmer config option.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/tplink.markdown [hunk 1]
```diff
@@ -27,6 +27,7 @@ There is currently support for the following device types within Home Assistant:
 
 - **Light**
 - **Switch**
+- **Dimmer**
```
Comment (klaasnicolaas) [inline source/_components/tplink.markdown]: Dimmer isn't a device type / (old) platform

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by rytilahti** [comment]
LGTM, approval from @klaasnicolaas is still required.

**Comment by klaasnicolaas** [comment]
This PR will be closed because the parent PR is closed 😉 Feel free to reopen it when there is new progress on the main repo.

**Comment by rytilahti** [comment]
The parent PR was merged some time ago (and the linked PR is about the missing documentation for `dimmers`) which is to be corrected with this PR, so I'm reopening it.

**Comment by klaasnicolaas** [comment]
Is this the right parent PR? https://github.com/home-assistant/home-assistant/pull/23668 
if yes, then this PR must get a rebase to the `Current` branch

**Comment by vangorra** [comment]
home-assistant/home-assistant#23668 is the correct PR. Looks like I merged incorrectly as my PR now includes commits from other people. Weird.

**Comment by klaasnicolaas** [comment]
It would have been better if you hadn't done that push because I could adjust the target branch. Now you will first have to remove the unrelated commits, you can do this by having your own branch perform a recent pull and push again to his PR.

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @vangorra!

This did not go totally according to plan I guess 😄 

Could you please rebase the PR and your commits onto the latest `current` branch to get rid of those unreleased commits? Thanks already! 👍

**Comment by vangorra** [comment]
There we go, fixed it.

**Review by frenck — APPROVED** [review]
Thanks for updating @vangorra! 👍

**Pull request merged**